buildSrc/private/src/main/kotlin/androidx/build/AndroidXComposeMultiplatformExtensionImpl.kt

```diff
@@ -121,3 +131,93 @@ open class AndroidXComposeMultiplatformExtensionImpl @Inject constructor(
             }
     }
 }
+
+fun Project.experimentalOELPublication() : Boolean = findProperty("oel.publication") == "true"
+fun Project.oelAndroidxVersion() : String? = findProperty("oel.androidx.version") as String?
+
+
+fun OELPublishingPrototype(project: Project) {
```

Comment:
Better name functions starting with verb (`configure`, for example)

Reply:
done